src/parser.c

```diff
@@ -1138,6 +1138,24 @@ static TSSymbol ts_alias_sequences[4][MAX_ALIAS_SEQUENCE_LENGTH] = {
   },
 };
 
+static inline bool sym_operator_identifier_character_set_1(int32_t lookahead) {
```

Comment:
These were just changes that happened when I ran `npm run build` which I'm assuming is due to the newer version of tree-sitter that was pulled down.